### DISCUSSION TITLE
What is the default version used during a GraphRequest?

### DISCUSSION THREAD

**Discussion by ivanngyb** [thebergamo/react-native-fbsdk-next#550] [Q&A] [answered]
Hi all, 

As the title suggests. What version does graph request default to if no version is provided?

Thanks

**Comment by hugoewarrior** [comment, marked as answer]
Hi @ivanngyb i was able to check the code and i found out that
1. For Android it will be based on what version of the SDK is installed in your Build gradle (com.facebook.android:facebook-android-sdk), in my case it was 17.0.2, then checking the release code, i found this file: [@JvmStatic fun getDefaultAPIVersion() = "v16.0"](https://github.com/facebook/facebook-android-sdk/blob/d56819b51007700d0ffc770905179162846e107f/facebook-core/src/main/java/com/facebook/internal/ServerProtocol.kt#L83) 
2. For IOS it is defined by the version on Podspec from this library (17.0) and then looking at the code we get: v17 https://github.dev/facebook/facebook-ios-sdk/blob/9ed465ccd73c231144d49a9c681774577c616f34/FBSDKCoreKit/FBSDKCoreKit/include/FBSDKCoreKitVersions.h#L10-L11


Hope it helps!

**Reply by ivanngyb** [reply]
Ah sweet, thanks! @hugoewarrior
